ghost-cat-app/src/app/View/image-details/image-details.component.ts

```diff
@@ -1,6 +1,9 @@
-import { Component, OnInit, Input, ViewChild, ElementRef } from '@angular/core';
+import { Component, OnInit, Input, ViewChild, ElementRef, ChangeDetectorRef } from '@angular/core';
 import { BoundingBoxModel } from 'src/app/Model/BoundingBoxModel';
 import { Output, EventEmitter } from '@angular/core';
+import { Shape } from 'src/app/Model/Shape';
+
+let drawableCanvas: boolean = false;
```

Comment:
Is there any reason that this was changed from a variable on the class to a global variable?

Reply:
I changed it because when I switched to a different image it wouldn't work... If I switched images and then enabled the drawable canvas it would still think it was disabled. I'm not quite sure why it was doing that but this seemed to fix the problem.